### DISCUSSION TITLE
点击+显示searching DOI然后就卡死没动静

### DISCUSSION THREAD

**Discussion by foredoom007** [MuiseDestiny/zotero-reference#284] [Q&A]
如题，单击+显示searching DOI再无响应。无法添加参考文献到文库并关联。

![微信图片_20240624104618](https://github.com/MuiseDestiny/zotero-reference/assets/59819217/b51886b0-e3a5-4188-a819-58669692e741)

**Comment by Garrick1texas** [comment]
请问贴主解决了吗

**Comment by QuinnYates** [comment]
同样的问题，DOI 其实已经获取成功了，就是没有办法import到我的目录里

**Reply by so11itude** [reply]
我也一样，你解决了吗
